goosebit/models.py

```diff
@@ -41,7 +61,9 @@ class Device(Model):
     feed = fields.CharField(max_length=255, default="default")
     flavor = fields.CharField(max_length=255, default="default")
     update_mode = fields.IntEnumField(UpdateModeEnum, default=UpdateModeEnum.ROLLOUT)
-    last_state = fields.CharField(max_length=255, null=True, default="unknown")
+    last_state = fields.IntEnumField(
+        UpdateStateEnum, null=True, default=UpdateStateEnum.UNKNOWN
```

Comment:
Why `null=True` here?  I think `UNKNOWN` is just the same as `None`, so we should decide which one we want.  My tendency is to lean towards `None`, but I think there might be a case for `UNKNOWN` such as updating it with any variant of this when the state information is lost or unparsable for some reason, but I still think `None` is reasonable.

Reply:
Correct. Changed.